### PR TITLE
Cma update tutorial template

### PR DESCRIPTION
**Pull request by OhadZivan**

### CODE REVIEW

File: tutorials/W1D2_Template/W1D2_Tutorial1.ipynb [hunk 1]
```diff
@@ -409,7 +410,11 @@
    },
    "outputs": [],
    "source": [
-    "# @title Video 1: Video 1 Name\n",
+    "# @title Video 1: Video 1 Name  # put in the title of your video\n",
+    "# note the libraries are imported here on purpose\n",
+    "\n",
+    "###@@@ for konstanine. a question, why isn't this above in the list of cells? \n",
+    "\n",
     "from ipywidgets import widgets\n",
```
Comment (iamzoltan) [inline tutorials/W1D2_Template/W1D2_Tutorial1.ipynb:418]: what do you mean exactly?

**Review by iamzoltan — COMMENTED** [review]
good additions! if the workflow completes successfully, we can merge and build

**Comment by iamzoltan** [comment]
It seems we are getting an error because the repo is not aware for the forked repos branch

**Comment by OhadZivan** [comment]
before the first cell, there is a list cells and their content.
the cell for video is not part of that list. it should be, as I believe all
days and tutorials should have a video.


--------------------------------------------------------------------------------------------
"Be who you are and say what you feel, because those who mind don't matter
and those who matter don't mind" ------ Dr. Seuss

**Comment by jlpearso** [comment]
@OhadZivan I believe that can be left out for the time being but could be nice for future revisions. The curriculum specialist will be in charge of putting videos into the tutorials before prepod so I think we'll be good for this year. They also have the examples from last years tutorials. Let's plan to take another pass over this when things are slower.

**Pull request merged**